tests/modules/restful_test/src/Plugin/resource/entity_test/EntityTests__1_0.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful_test\Plugin\resource\entity_test\EntityTests__1_0.
+ */
+
+namespace Drupal\restful_test\Plugin\resource\entity_test;
+
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+
+/**
+ * Class EntityTests__1_0
+ * @package Drupal\restful_test\Plugin\resource
+ *
+ * @Resource(
+ *   name = "entity_tests:1.0",
+ *   resource = "entity_tests",
+ *   label = "Entity tests",
+ *   description = "Export the entity test with multiple bundles.",
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "entity_test",
+ *     "bundles": {
+ *       "test",
+ *       "main"
+ *     },
+ *   },
+ *   majorVersion = 1,
+ *   minorVersion = 0
+ * )
+ */
+class EntityTests__1_0 extends ResourceEntity implements ResourceInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function publicFields() {
+    $public_fields = parent::publicFields();
+
+    $public_fields['main_bundle'] = array(
```

Comment:
I see, JSON API is indeed cleaning it up nicely :) 

but just to see that I understand, could you please show how would `simple json` formatter look.